app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,12 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+.ideas.listing {
+  margin-top:20px;
+}
+
+.favorite-icon {
+  &:hover {
+    cursor: pointer;
+    g polygon {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3